go/service/teams.go

```diff
@@ -420,6 +420,28 @@ func (h *TeamsHandler) TeamEditMember(ctx context.Context, arg keybase1.TeamEdit
 	return teams.EditMember(ctx, h.G().ExternalG(), arg.Name, arg.Username, arg.Role, arg.BotSettings)
 }
 
+func (h *TeamsHandler) TeamEditMembers(ctx context.Context, arg keybase1.TeamEditMembersArg) (res keybase1.TeamAddMembersResult, err error) {
+	ctx = libkb.WithLogTag(ctx, "TM")
+	debugString := "0"
+	if len(arg.Users) > 0 {
+		debugString = fmt.Sprintf("'%v, %v'", arg.Users[0].AssertionOrEmail, arg.Users[0].Role)
+		if len(arg.Users) > 1 {
+			debugString = fmt.Sprintf("'%v' + %v more", arg.Users[0].AssertionOrEmail, len(arg.Users)-1)
+		}
+	}
+	defer h.G().CTraceTimed(ctx, fmt.Sprintf("TeamEditMembers(%s, %s)", arg.Name, debugString),
+		func() error { return err })()
+	if len(arg.Users) == 0 {
+		return res, fmt.Errorf("attempted to add 0 users to a team")
```

Comment:
Change "attempted to add 0 users to a team" to "attempted to change the role of 0 users". But actually I think it'd be better to just succeed.